apps/sanity/schemas/shop.ts

```diff
@@ -20,8 +20,8 @@ export default {
       type: 'geopoint',
     },
     {
-      title: 'Shop Logo',
-      name: 'logo',
+      name: 'logoUrl',
```

Comment:
Please double check this, so we will not accidentally change `logo` to `logoUrl`.